src/export/ValueSetExporter.ts

```diff
@@ -49,11 +50,19 @@ export class ValueSetExporter {
           composeElement.system =
             this.fisher.fishForMetadata(component.from.system, Type.CodeSystem)?.url ??
             component.from.system;
+          if (!isUri(composeElement.system)) {
+            throw new InvalidUriError(composeElement.system);
+          }
         }
         if (component.from.valueSets) {
           composeElement.valueSet = component.from.valueSets.map(vs => {
             return this.fisher.fishForMetadata(vs, Type.ValueSet)?.url ?? vs;
           });
+          composeElement.valueSet.forEach(vs => {
+            if (!isUri(vs)) {
```

Comment:
Technically, `compose.include.valueSet` is type `canonical(ValueSet)`.  And according to FHIR doc:
> The canonical type differs from a uri in that it has special meaning in this specification, and in that it may have a version appended, separated by a vertical bar (|).

So... we need to support `|` here, but `isUri` rejects it since URIs typically don't allow `|`.

Reply:
Oh FHIR how tricky of you. I will fix this.